scripts/script_sync.py

```
@@ -8,10 +8,11 @@
 import subprocess
 import sys
 import os
+import shutil
 from pathlib import Path
-from typing import List, Set
+from typing import List, Optional, Set
```

Flake8 (7.3.0)

[error] 'typing.Set' imported but unused

(F401)